### PR TITLE
Couple of cephmetric fixes

### PR DESCRIPTION
**Pull request by b-ranto**
This contains a fix for ansible where we start grafana way too early (before we modify its config file), adds rpm support for piechart plugin and fixes collectors constructors so that they support log_level keyword argument.

### CODE REVIEW

**Comment by zmc** [comment]
I think we should use https://github.com/ceph/cephmetrics/pull/29 instead of 2a6f76d

**Comment by b-ranto** [comment]
Nope, that won't work, notify behaves as if it was async, it only guarantees that the service will be restarted at some point in the future (when I was testing it, it restarted the service way too late).

**Comment by zmc** [comment]
@b-ranto that's what [`meta: flush-handlers`](https://docs.ansible.com/ansible/meta_module.html#options) is for

File: ansible/roles/ceph-grafana/tasks/configure_grafana.yml [hunk 1]
```diff
@@ -1,10 +1,45 @@
...
+  service:
+    name: grafana-server
+    state: stopped
+
+- name: Wait for grafana to be stopped
+  wait_for:
+    port: 3000
+    state: stopped
+
+- set_fact:
```
Comment (zmc) [inline ansible/roles/ceph-grafana/tasks/configure_grafana.yml]: Stray `set_fact` call

Comment (b-ranto) [inline ansible/roles/ceph-grafana/tasks/configure_grafana.yml]: good catch, should be fixed, now.

Comment (zmc) [inline ansible/roles/ceph-grafana/tasks/configure_grafana.yml]: Hmm, doesn't appear to have updated

Comment (b-ranto) [inline ansible/roles/ceph-grafana/tasks/configure_grafana.yml]: Yeah, github seems to be a bit slow today but the updated patch will hopefully show up soon.

**Pull request merged**